plugins/org.jkiss.dbeaver.ext.postgresql/src/org/jkiss/dbeaver/ext/postgresql/model/PostgreSchema.java

```diff
@@ -109,6 +110,11 @@ public class PostgreSchema implements
         dataTypeCache = new PostgreDataTypeCache();
     }
 
+    @Override
+    public List<DBSObjectContainer> getPublicContainers(@NotNull DBRProgressMonitor monitor) throws DBException {
+        return List.of(this.database.getSchema(monitor, PostgreConstants.INFO_SCHEMA_NAME));
```

Comment:
I think this logic must be somewhere in AbstractDataSource because info schema is SQL standard (it even present in some NoSQL databases). We can add something like isInformationSchemaSupported in DBPDataSourceInfo.

In PostgreSQL pg_catalog is also always visible.

Reply:
In Oracle SYS and SYSTEM are public schemas.
In DB2 SYS and IBMSYS and maybe something else too.

Reply:
I separated the public schemas support from this PR. Will handle it in scope of #[35430](https://github.com/dbeaver/dbeaver/issues/35430) and https://github.com/dbeaver/dbeaver/issues/35313#issuecomment-2314762724